frontend/pages/media/marketing.js

```diff
@@ -0,0 +1,233 @@
+import React, { useEffect, useState } from "react";
+import styles from "../../styles/Marketing.module.css";
+import PageHeader from "../../components/Header";
+import Chip from "@material-ui/core/Chip";
+import Initiative from "../../components/Initiative";
+import CircularProgress from "@mui/material/CircularProgress";
+import Timeline from "../../components/Timeline";
+import PaginationComp from "../../components/Pagination";
+import LoadingScreen from "../../components/LoadingScreen";
+import axios from "axios";
+import humps from "humps";
+import { isMobile } from "react-device-detect";
+import {
+  useStyles,
+  categories,
+  marks,
+  valueToYear,
+} from "../../data/marketingData";
+
+const MarketingContent = () => {
+  const classes = useStyles();
```

Comment:
Update the use states/general code of this page - Kesh's feature has fixed the timeline bug ie category defaults to all after switching year and content that was not showing up for some categories (eg special occasions), now shows up!